displays/cavity_display/backend/fault.py

```diff
@@ -73,12 +72,13 @@ def __init__(
 
         # Storing PV name as a string instead of making a PV obj
         self.pv: str = pv
-        self._pv_obj: Optional[PV] = None
+        # TODO figure out why lazy generation breaks the backend runner
+        self._pv_obj: Optional[PV] = PV(self.pv) if not lazy_pv else None
 
     @property
     def pv_obj(self) -> PV:
         if not self._pv_obj:
-            self._pv_obj = PV(self.pv, connection_timeout=PV_TIMEOUT)
```

Comment:
Why don't we need the connection_time anymore for PVs? I noticed we decided not to include a connection_timeout for all the new PV() things in this pull request

Reply:
I realized it was redundant; the default timeout time is already that value lol

Reply:
Oh, nice, that's convenient!